addons/mrp/models/stock_orderpoint.py

```diff
@@ -42,6 +42,21 @@ def _compute_show_bom(self):
         for orderpoint in self:
             orderpoint.show_bom = orderpoint.route_id.id in manufacture_route
 
+    def _get_min_qty_to_order(self):
+        """ Calculates the minimum quantity that can be ordered according to the qty and UoM of the BoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_to_order()
+        if min_qty_order_possible:
+            return min_qty_order_possible
+        elif self.route_id:
+            rules_actions = self.route_id.rule_ids.mapped('action')
+            boms = self.product_id.bom_ids.filtered(lambda b: b.type == 'normal')
+            if 'manufacture' in rules_actions and boms:
+                bom = boms[0]
+                min_qty_order_possible = bom.product_uom_id._compute_quantity(bom.product_qty, self.product_uom)
+                return min_qty_order_possible
```

Comment:
You should return anyway (outside double if) ? return sometime False, and sometime None.

Reply:
It should be homogenized indeed. 
I guess it's simpler to change the default return as None.